src/global_route_configurator.ts

```diff
@@ -30,11 +29,23 @@ export async function configureGlobalRoutes(container: InvocationContainer): Pro
 }
 
 function configureRootRoute(): void {
+
+  const allowUseOfGlobalRoute = process.env.DO_NOT_BLOCK_GLOBAL_ROUTE === undefined;
```

Comment:
So if DO_NOT_BLOCK_GLOBAL_ROUTE is set to false it will still not use them? 😄

I see no reason to do this, but shouldn't it be handled anyway?

Reply:
Well basically, this is suppsed to be a flag. Like "it exists, or it doesn't", regardless of its value. But I can see how this might appear a little strange. 😉 

```suggestion
  const allowUseOfGlobalRoute = !process.env.DO_NOT_BLOCK_GLOBAL_ROUTE;
```